base_classes/NXdetector.nxdl.xml

```diff
@@ -470,6 +462,8 @@
             </doc>
             <!--. For a pixel detector it provides the nominal wavelength-->
             <!--. For other detectors this field has to be seen together with-->
+            <!--. For a pixel detector it provides the nominal wavelength-->
```

Comment:
is the comment duplicated?

Reply:
@sanbrock, There should not be any comments at all, this comments are coming from doc exactly on top of the comment blocks, where the order of the paragraphs was addressed by '#'. Therefore the paragraphs have been considered as comments by 'nyaml2nxdl' converter.

I fixed it by removing '#' by '1, 2, ...' from the order of paragraphs.